data/interfaces.txt

```diff
@@ -144,6 +144,18 @@ interface "main menu"
 		center -420 155
 		dimensions 120 30
 
+	# For Beta Only
+	label "THIS IS THE BETA VERSION"
```

Comment:
Maybe make this "AN UNSTABLE VERSION" rather than "BETA VERSION" so as not to imply that the stable release is non-beta. It still is, since 1.0 is not out.

Reply:
Just changed it over to Unstable.  Good catch; was thinking of the "Beta" branch on Steam I guess.